src/styles/base.css

```diff
@@ -873,3 +873,19 @@ a.card:hover {
 #pluginPage #grid-box .content .main td {
   padding: 1em;
 }
+
+#pluginPage .plugin-governance-notice {
+  padding: 5px;
+  background-color: #fcf8e3;
+  color: #a48d62;
+  border-color: #faebcc;
+  margin-bottom: 5px;
```

Comment:
I think you would be better served adding a `margin-top: 10px`

and adding a `margin-top: 10px` to `content` class 
https://github.com/jenkins-infra/plugin-site/blob/5e62e6f8b51643344875c0ba52de07d33adc64a7/src/templates/plugin.jsx#L68

(feel free to mess around with the pixel number)

Reply:
after adding 10px to content:
![image](https://user-images.githubusercontent.com/1661688/74857058-6f185000-5343-11ea-8f94-c7d5f4e4d72d.png)

Reply:
I plan to rework the  dependency and maintainer listing soon, so I do not worry that much about margins for now.  I would suggest doing it as a separate change

Reply:
fine 😅